R/repl.R

```diff
@@ -14,7 +14,7 @@
 #' @export
 py_repl <- function(
   module = getOption("reticulate.repl.module", default = character()),
```

Comment:
I think this should default to `NULL` (with no option) just to make the interface more straightforward. We can add an R option if we discover that it's useful, but I don't want to start with this.